apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -246,10 +250,49 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
 
             return roles;
         }, {});
-    
+
         setRoleUserAssociations(newRoles);
     }, [ rolesList ]);
 
+    /**
+     * Function to fetch and update group list for a given domain
+     */
+    const getGroupListForDomain = (domain: string) => {
+        getGroupList(domain)
```

Comment:
Hope we don't have an SWR hook for `Group List` @dasuni-30

Reply:
Improved with b27e3b23782e4d15df66aed605a31adfc2415a4e.